From 6164408188766a31e778ec670239482db886d9be Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sun, 13 Nov 2022 14:41:08 +0900
Subject: [PATCH] bootctl: downgrade log message when firmware reports
 non-existent or invalid boot entry

Fixes #25359.

(cherry picked from commit 78bfeeae508a554483de02b52aa2e5afdc341e1a)

Related: #2138081
---
 src/boot/bootctl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/boot/bootctl.c b/src/boot/bootctl.c
index 64a4b74715..a5811adf20 100644
--- a/src/boot/bootctl.c
+++ b/src/boot/bootctl.c
@@ -586,7 +586,7 @@ static int print_efi_option(uint16_t id, int *n_printed, bool in_order) {
 
         r = efi_get_boot_option(id, &title, &partition, &path, &active);
         if (r < 0)
-                return log_error_errno(r, "Failed to read boot option 0x%04X: %m", id);
+                return log_debug_errno(r, "Failed to read boot option 0x%04X: %m", id);
 
         /* print only configured entries with partition information */
         if (!path || sd_id128_is_null(partition)) {
